week-10/10-tech-lab.adoc

```diff
@@ -0,0 +1,228 @@
+
+You will research the results and recommend at least one fix that can be implemented.
+
+You will fix at least two vulnerabilities and participate in group and class discussion about how this relates to the broader topics of governance, risk, and compliance.
+
+
+=== Getting started
+
+Start up a Virtual Machine, login to your personal server, perform an update and install lynis.
+Lynis will not be installed in the directory you are currently in.  Use the 'find' command to determine where it was installed.
+Once you know where it got downloaded, run 'audit system' with the --quick option (don't wait for user input).  Lynis will generate hundreds
+of findings that will scroll off your display.  Redirect the lynis output to a file, e.g. lynis_out, so you can view it later.
+
+....
+cd ~
+sudo apt-get update
+sudo apt-get install lynis
+sudo find / -name lynis
+....
+
+Examine the locations to find the executable lynis*.  Once you know where it got downloaded, run 'audit system' with the --quick option (don't wait for user input).  
+Lynis will generate hundreds of findings that will scroll off your display.  Redirect the lynis output to a file, e.g. lynis_out, so you can view it later.
+
+....
+sudo <lynis* directory>/lynis --quick audit system > lynis_out
+....
+
+The lynis scan is run internally on the server, as a script or application that reviews the server's configuration in painstaking detail.   
+It will generate hundreds of findings.  You should pick one or two and research them and figure out if there is an easy fix.
+
+This scan is run internally on the server, as a script or application that reviews the 
+server's configuration in painstaking detail. Lots of output that looks like:
+
+....
+- Checking for old files in /tmp                            [ OK ]
+ - Checking /tmp sticky bit                                 [ OK ]
+ - Mount options of /                                       [ OK ]
+ - Checking Locate database                                 [ FOUND ]
+.
+.
+.
+
+ * Consider hardening SSH configuration [SSH-7408]
+    - Details  : AllowAgentForwarding (YES --> NO)
+      https://cisofy.com/controls/SSH-7408/
+
+  * Configure minimum password age in /etc/login.defs [AUTH-9286]
+      https://cisofy.com/controls/AUTH-9286/
+
+  * Configure maximum password age in /etc/login.defs [AUTH-9286]
+      https://cisofy.com/controls/AUTH-9286/
+....
+
+Let's fix the minium and maxium password age issue in login.defs.
+
+Open file /etc/login.defs using text editor:
+# sudo nano /etc/login.defs
+
+Setup (sample) values as follows:
+
+ PASS_MAX_DAYS 30
+ PASS_MIN_DAYS 1
+ PASS_WARN_AGE 7
+
+To restart sshd on ubuntu 16, use the command:
+
+ sudo systemctl restart sshd
+
+For earlier versions of Ubuntu, use:
+
+ sudo service ssh restart
+
+Between these two resources you should be able to fix the issue. If you have questions, post them to Slack. When you have fixed the issue, run the scan again and confirm the banner is no longer an issue.
```

Comment:
I can't remember which "two resources" I was thinking of. PLease rewrite. Also we don't use Slack. That's the kind of thing I need you to clean up.